packages/terra-docs/src/Notice.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+
+import styles from './Notice.module.scss';
+
+const cx = classNames.bind(styles);
+
+const getTitle = (variant) => {
+  if (variant === 'deprecation') {
+    return 'Deprecation Notice';
+  }
+
+  return 'error';
+};
+
+const propTypes = {
+  /**
+  * The main body text of the notice.
+  */
+  children: PropTypes.string,
+  /**
+  * The type of notice.
+  */
+  variant: PropTypes.oneOf(['deprecation']),
```

Comment:
I added variants assuming we may want to color coordinate and keep consistent titles for our notices. So far i've only added a deprecation notice. Other ideas may be noting an 'expert level' component or warning a user that they may want to use a simpler component in stead?

Reply:
I think a variant of 'maintenance' may make sense. a banner notifying that this component is not recommended but still supported until some future date.